cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
```

Comment:
The socket can be seen from the host, and other pods could mount the host directory with the socket and talk to libvirt. We can't influence what non-KubeVirt workloads are doing.